src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/activitiesappointments/ActivitiesAppointmentsClient.kt

```diff
@@ -83,4 +86,22 @@ class ActivitiesAppointmentsClient(private val activitiesAppointmentsApiWebClien
       .bodyToMono(typeReference<List<AppointmentSearchResult>>())
       .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
       .block() ?: emptyList()
+
+  /**
+   * @param appointmentId refers the appointment identifier held in Activities and Appointments, not BVLS.
+   */
+  fun cancelAppointment(appointmentId: Long) {
```

Comment:
We have to be 100% here we have the right appointments when deleting.  So needs testing thoroughly.